File: RokniAppApi/aspnet-core/src/RokniAppApi.Domain/Stock/Stock.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using RokniAppApi.Domain.NoteModel;
using RokniAppApi.IndustryModel;
using Volo.Abp.Domain.Entities.Auditing;

namespace RokniAppApi.StockModel
{
  public class Stock : AuditedAggregateRoot<Guid>
  {
    public string Name { get; set; }
    public string ChartIndex { get; set; }
    public string CodalLink { get; set; }
    public string ChartsazLink { get; set; }
    public string TablokhaniLink { get; set; }
    public string TsetmcLink { get; set; }
    public string RahvardLink { get; set; }
    public string ShakhesbanLink { get; set; }
    public bool Choosen {get; set;}
    
    public string ChartixLink { get; set; }
    public int SortNumber { get; set; }
    public Guid IndustryId { get; set; }
    public Guid StockNotebookId { get; set; }
    public virtual StockNotebook StockNotebook { get; set; }
    public virtual Industry Industry { get; set; }

  }
}

File: AppUI/src/app/Models/Stock.ts
import { IndustryDto } from "./Industry";
import { StockNotebookDto } from "./Notebook";

export interface StockDto {
  id: string,
  name: string,
  codalLink: string,
  chartsazLink: string,
  tablokhaniLink: string,
  tsetmcLink: string,
  chartIndex: string,
  rahvardLink: string,
  shakhesbanLink: string,
  chartixLink:string,
  industryId: string,
  choosen:boolean,
  stockNotebook: StockNotebookDto,
  stockNotebookId: string,
  industry: IndustryDto
}


export interface StockCreateUpdateDto {
  id?: string,
  name: string,
  codalLink: string,
  chartsazLink: string,
  chartIndex: string,
  tablokhaniLink: string,
  tsetmcLink: string,
  rahvardLink: string,
  shakhesbanLink: string,
  chartixLink:string,
  industryId: string,
  sortNumber: number,
  choosen:boolean,
  stockNotebookId: string
}